funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -272,8 +272,12 @@ def __init__(
             self.worker_task_port = self.worker_ports[0]
             self.worker_result_port = self.worker_ports[1]
 
-            self.task_outgoing.bind(f"tcp://*:{self.worker_task_port}")
-            self.results_incoming.bind(f"tcp://*:{self.worker_result_port}")
+            self.task_outgoing.bind(
+                f"tcp://{self.interchange_address}:{self.worker_task_port}"
+            )
+            self.results_incoming.bind(
+                f"tcp://{self.interchange_address}:{self.worker_result_port}"
+            )
```

Comment:
This appears to only be half of the equation.  That is, the subprocess interchange.  I think we'll also need to look at `HighThroughputExecutor.start()` which has this suspicious code:
```python
# funcx_endpoint.executors.high_throughput.executor.HighThroughputExecutor.start
def start(
    self,
    results_passthrough: multiprocessing.Queue = None,
    funcx_client: FuncXClient = None,
):
    """Create the Interchange process and connect to it."""
    self.outgoing_q = zmq_pipes.TasksOutgoing(
        "0.0.0.0", self.interchange_port_range
    )
    self.incoming_q = zmq_pipes.ResultsIncoming(
        "0.0.0.0", self.interchange_port_range
    )
    self.command_client = zmq_pipes.CommandClient(
        "0.0.0.0", self.interchange_port_range
    )
    ...
```
As an aside, reading through the code suggests to me that open these _prior_ to forking, so the child processes also have these file descriptors open.  Hmm.

Reply:
@khk-globus Sorry, I should've pushed my changes yesterday night itself. I had this fixed on Frontera but hadn't finished up testing. The new changes should limit the interfaced used for all communication on the endpoint side. I did some testing with `lsof -i` to confirm.